### DISCUSSION TITLE
Download titles that dont require borrowing

### DISCUSSION THREAD

**Discussion by atlury** [elementdavv/internet_archive_downloader#53] [Q&A]
I understand that the download button appears when borrowed but is it possible to download for books that dont require borrow (copy-right free). IA pdfs for such  books have bad compression.

Yours has an option to download high quality images stitched together.

**Comment by elementdavv** [comment]
There is no plan to work with always available books for the extension.

**Comment by atlury** [comment]
Aah thanks for that! May be I will explore it myself when I get some free time.
